src/rdb/rdb_tx.c

```diff
@@ -397,6 +381,64 @@ rdb_tx_append(struct rdb_tx *tx, struct rdb_tx_op *op)
 	return 0;
 }
 
+/**
+ * Commit \a tx. If successful, then all updates in \a tx are revealed to
+ * queries. If an error occurs, then \a tx is aborted.
+ *
+ * \param[in]	tx	transaction
+ *
+ * \retval -DER_NOTLEADER	this replica not current leader
+ */
+int
+rdb_tx_commit(struct rdb_tx *tx)
```

Comment:
It would be nice if we could avoid moving rdb_tx_commit. For example, by adding a declaration of rdb_tx_is_critical before rdb_tx_commit.

Reply:
Discussed with @liw that this change will not be included in this patch.